modules/ROOT/pages/salesforce/salesforce-connector-96.adoc

```diff
@@ -0,0 +1,1211 @@
+the pom.xml for your project:
+
+[source,xml,linenums]
+----
+<repositories>
+   <repository>
+        <id>mule-ee-releases</id>
+        <name>MuleEE Releases Repository</name>
+        <url>https://repository-master.mulesoft.org/nexus/content/repositories/releases-ee/</url>
+    <repository>
+        <id>mule-ee-snapshots</id>
+        <name>MuleEE Snapshots Repository</name>
+        <url>https://repository-master.mulesoft.org/nexus/content/repositories/ci-snapshots/</url>
+    </repository>
+</repositories>
+----
+
+This snippet specifies a project dependency for a specific release version:
+
+[source,xml,linenums]
+----
+<dependency>
+    <groupId>org.mule.connectors</groupId>
+    <artifactId>mule-sfdc-connector</artifactId>
+    <version>RELEASE</version>
+    <classifier>mule-plugin</classifier>
+</dependency>
+----
+
+This snippet adds specifies a project dependency for the latest version:
+
+[source,xml,linenums]
+----
+<dependency>
+    <groupId>org.mule.modules</groupId>
+    <artifactId>mule-sfdc-connector</artifactId>
+    <version>LATEST</version>
+</dependency>
+----
+
+== Configure in Design Center
+
+. In Anypoint Platform > Design Center, click Create > Mule Application.
+. Click a trigger to initiate access to your flow. To have Salesforce send content to your application, select Salesforce as the trigger and an operation:
++
+* *New Object* - Create a new object. Possible object types are Contact, Lead, Account, Contracts, Opportunity, and custom objects. The object types correspond to the Salesforce object you want to create, modify, or delete.
+* *Modified Object* - Modify an object.
+* *Deleted Object* - Delete an object.
+* *Replay Channel* - Subscribe to a streaming channel. This trigger provides channel-related notification settings for new events and events stored by Salesforce within the 24-hour retention period. Events that take place in the specified channel trigger the flow.
++
+Note: A channel or topic must be published to Salesforce before a subscription to the channel can be created.
++
+* *Replay Topic* - Subscribe to a topic. This trigger provides topic-related notification settings for new events and events stored by Salesforce within the 24-hour retention period. Events that take place on records in the specified topic trigger the flow.
+* *Subscribe Channel* - Subscribe to a streaming channel. This trigger provides channel-related notification settings for new events that take place after you subscribe.
+* *Subscribe Topic* - Subscribe to a topic. This trigger provides topic-related notification settings for new events that take place after you subscribe.
++
+Alternatively, you can use an HTTP Listener or Scheduler trigger.
+
+== Configure a Salesforce New Object Trigger
+
+The following shows how to configure the Salesforce New Object trigger.
+
+. Create a New Mule Application in Design Center.
+. Set the trigger. For example, you can use Salesforce > New Object.
```

Comment:
UI text needs bold throughout. I'll stop marking it, but will happily re-review.